### PR TITLE
Make UrlInputFragment Locale Aware #1066

### PR DESCRIPTION
**Pull request by ekager**

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
 /**
  * Fragment for displaying he URL input controls.
  */
-public class UrlInputFragment extends Fragment implements View.OnClickListener, InlineAutocompleteEditText.OnCommitListener, InlineAutocompleteEditText.OnFilterListener, PopupMenu.OnMenuItemClickListener {
+public class UrlInputFragment extends LocaleAwareFragment implements View.OnClickListener, InlineAutocompleteEditText.OnCommitListener, InlineAutocompleteEditText.OnFilterListener, PopupMenu.OnMenuItemClickListener {
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: The class 'UrlInputFragment' has a Cyclomatic Complexity of 4 \(Highest = 25\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: The class 'UrlInputFragment' has a Modified Cyclomatic Complexity of 3 \(Highest = 25\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: The class 'UrlInputFragment' has a Standard Cyclomatic Complexity of 4 \(Highest = 25\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: This class has too many methods, consider refactoring it\.

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java [hunk 2]
```diff
@@ -69,6 +69,14 @@ public static UrlInputFragment createWithBackground() {
         return fragment;
     }
 
+    @Override
+    public void applyLocale() {
+        getActivity().getSupportFragmentManager()
+                .beginTransaction()
+                .replace(R.id.container, UrlInputFragment.createWithBackground(), UrlInputFragment.FRAGMENT_TAG)
```
Comment (pocmo) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: I wonder if "withBackground" is always correct here? What happens if we show this as an overlay and I leave the app - go to the system settings, change the locale and resume the app?

Comment (ekager) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: I added a check for isOverlay !

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java (hunk 1 above)
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: The class 'UrlInputFragment' has a Cyclomatic Complexity of 4 \(Highest = 25\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: The class 'UrlInputFragment' has a Modified Cyclomatic Complexity of 3 \(Highest = 25\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: The class 'UrlInputFragment' has a Standard Cyclomatic Complexity of 4 \(Highest = 25\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: This class has too many methods, consider refactoring it\.

**Pull request merged**